File: src/main/java/dev/sky_lock/mocar/commands/DebugCommand.java
package dev.sky_lock.mocar.commands;

import dev.sky_lock.mocar.gui.EditModelData;
import dev.sky_lock.mocar.gui.EditSessions;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

/**
 * @author sky_lock
 */

public class DebugCommand implements ICommand, IAdminCommand {

    @Override
    public void execute(CommandSender sender, Command cmd, String[] args) {
        EditModelData data = EditSessions.get(((Player) sender).getUniqueId());
        if (data == null) {
            sender.sendMessage("hogehoge");
            return;
        }
        sender.sendMessage(data.getId() + " : " + data.getName());
    }
}

File: src/main/kotlin/dev/sky_lock/pocketlifevehicle/command/DebugCommand.kt
package dev.sky_lock.pocketlifevehicle.command

import dev.sky_lock.pocketlifevehicle.vehicle.VehicleManager
import org.bukkit.Bukkit
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

/**
 * @author sky_lock
 */
class DebugCommand : ICommand, IAdminCommand {
    override fun execute(sender: CommandSender, cmd: Command, args: Array<String>) {
        val vehicleMap = VehicleManager.vehicleMap
        vehicleMap.forEach { uuid, vehicle ->
            val player = Bukkit.getPlayer(uuid) ?: return@forEach
            Bukkit.broadcastMessage(player.name + " : " + vehicle.model.id)
        }
    }
}